src/rpcmining.cpp

```diff
@@ -415,8 +416,14 @@ Value getblocktemplate(const Array& params, bool fHelp)
 
     LOCK(cs_main);
 
+    // Wallet is required because we support coinbasetxn
+    if (pwalletMain == NULL) {
+        throw JSONRPCError(RPC_METHOD_NOT_FOUND, "Method not found (disabled)");
+    }
```

Comment:
Currently, there seems to be an issue building zcash with wallet disabled - e.g. here: https://github.com/zcash/zcash/issues/1618

For integration at slushpool, we would like to use a fixed coinbase address and operate the mining nodes without wallets. Will this be supported? As of now, the build fails as indicated by the above issue.